packages/labs/virtualizer/src/layouts/flow.ts

```diff
@@ -312,6 +312,8 @@ export class FlowLayout extends BaseLayout<BaseLayoutConfig> {
     } else {
       this._getItems();
     }
+    // Fix scroll size after rendering
+    this._updateScrollSize();
```

Comment:
With items having different sizes, end of scrolling container was not aligned with bottom of last item. I needed to had this update scroll size, with code update in `_updateScrollSize`